teradatalabs/cdh5-hive-kerberized/files/conf/yarn-site.xml

```diff
@@ -15,7 +15,6 @@
   See the License for the specific language governing permissions and
   limitations under the License.
 -->
-<?xml-stylesheet type="text/xsl" href="configuration.xsl"?>
```

Comment:
`<?xml>` and `<?xml-stylesheet>` are different. Maybe you could merge this nodes into single `<?xml stylesheet..>`